certificate-policy.md

```diff
@@ -1089,6 +1095,52 @@ The business continuity plan MUST include:
+- A known or reasonably known, publicly reported compromise of Certificate Systems
+- Any other issue that the FPKIPA identifies as calling into question the CAs integrity or trustworthiness
+
+In the event of a CA or certificate compromise or fraudulent mis-issuance, the CA shall notify the FPKIPA as soon as possible, but no later than 24 hours from the time the incident was discovered. An initial security incident report shall be submitted to the FPKI@GSA.gov email or communicated directly to the FPKIPA and include the following sections:
+1. Which Certificate Systems or components were affected by the incident
+2. The CA's interpretation of the incident.
+3. Was the incident detected as part of normal operations. If not, explain why.
+4. Who detected the incident or perpetrated if known
+5. When the incident was discovered
+6. Physical location of the incident, if applicable.
+7. A partial or complete list of all certificates that were either mis-issued or not compliant with the CP/CPS as a result of the incident.
+
+A final security incident report shall be submitted at a date specified by the FPKIPA to the same location as the initial incident report and include all sections identified below.
+1. A complete timeline of events.
+2. If a compromise, a detailed description of the exploit and what and how infrastructure was compromised.
+3. If the CA did not detect the incident, why not.
+4. What specific remedial measures were taken or will take to address the underlying cause including specific CP/CPS updates.
+5. Other information appropriate to understand the incident such as system or vendor documentation or other material.
+6. Proof the mis-issued certificates were revoked.
+7. Who detected or perpetrated the incident.
+8. If requested, log files.
+9. Detailed description of how the incident was closed.
+
+In coordination with the CA, the FPKIPA may conduct the following activities as part of an incident response.
+- Communicate with affected parties or directly with affected organizations
+- Publish notice of revocation
+- Publicly publish a final security incident report on an approved government website.
+- Require the CA to employ, at the CA expense, a third party investigator to investigate the security incident and prepare a final security incident report.
+- Request specific reports at a periodic interval as determined by the FPKIPA
+- Specify a due date for the CA to submit a final security incident report.
+
+The FPKIPA shall notify the CA, in writing, of its intentions in response to the security incident seven (7) days prior to the action by the FPKIPA except under exceptional circumstances (as defined in the glossary) where the FPKIPA will make reasonable efforts to communicate with the CA prior to taking action. The  CA may propose an alternate course of action and the FPKIPA may consider reasonable alternatives but reserves the right to reject any proposed course of action not in the government’s best interest.
+
```

Comment:
So, this seems like a lot to put into the CP, and it's blurring some lines on what the CP is describing. For example, I don't think the FPKIPA's obligations on timeliness in responding to security incident reports is within the bounds of what the CP should describe (and what we wish to be audited on).